wellcomecollection/wellcomecollection.org
# Update SVGR

Relates to #11040

## What does this change?

Updates [SVGR](https://react-svgr.com/) to the most recent version in order to stop depending on a version of `minimist` that is responsible for a [critical vulnerability](https://github.com/wellcomecollection/wellcomecollection.org/security/dependabot/80).

I've changed the output of the template to be more aligned with the docs (e.g. exporting a `function ComponentName(props) {...}` rather than `const ComponentName = (props) => {...}`.

## How to test

```console
$ cd common/icons && yarn compile
```

`yarn tsc` at the root to verify TypeScript is happy

`yarn cardigan` and visit the [icons page](http://localhost:9001/?path=/story/global-icons--page) to see they're still rendering

Maybe run `yarn content` to check you can see some icons (e.g. in the footer)

## How can we measure success?

We remove a critical vulnerability

## Have we considered potential risks?

Can't think of any.
